src/Form.jsx

```diff
@@ -291,12 +291,14 @@ function Form({
                     servar.type === 'phone_number' &&
                     servar.metadata.send_sms_code
                 ) {
+                    let setCaptcha = false;
                     newStep.buttons.forEach((button) => {
```

Comment:
I'd recommend `newStep.buttons.find(b => b.link === 'submit')` and then doing logic based on whether the return value is defined

Reply:
makes sense